makefile

```diff
@@ -15,3 +15,6 @@ build:
 
 test:
 	npm run test
+
+snapshots.test.js:
```

Comment:
this should be the full path i.e. `__tests__/__snapshots__/snapshots.test.js.snap`, and the documentation should reflect this

Reply:
(or, add an `update-snapshots: __tests__/__snapshots__/snapshots.test.js.snap` alias task)

Reply:
hmm, second (third?) thoughts, doing the actual file path with no prerequisites means the snapshots will never actually update, make will see the file is there and tell you it's up to date. so we need to either:

- mark it as `.PHONY`
- add prerequisites (probably something like `story-files = $(wildcard components/{stories,src}/{**,}/*.{js,jsx}` and `__tests__/__snapshots__/snapshots.test.js.snap: $(story-files)`)
- just use a generic task name like before

Reply:
OK, I'm guessing I'm not the only one who is not very familiar with makefiles.

My vote would be to go back to the generic `update-snapshots` task name, as that's consistent with how we're using `make` already in this repo and elsewhere, and it's easy to understand. I don't feel strongly though.

Reply:
i'd love to get `n-gage` to a point where it can _make_ (a-hyuck) this kind of thing easy, turn it into more of a framework for writing makefiles instead of the giant blob of whatever it is currently